### ISSUE TITLE
Remove all of the build.release.yaml, use --release

### ISSUE THREAD

**Issue by kevmoo** [dart-lang/angular#1213]
1) Set min dependency on build_runner to 0.8.2 and build_web_compilers to 0.3.6
2) Configure dart2js in build.yaml iff you want special config https://github.com/dart-lang/build/blob/master/docs/getting_started.md#switching-to-dart2js
   otherwise, do onthing
3) Set up any auto-build to pass in `--release` instead of `--config release`
4) Delete `build.release.yaml`

**Comment by kevmoo** [comment]
CC @natebosch @jakemac53

**Comment by matanlurey** [comment]
Unless these are the default `dart2js_args`, we cannot do that:

```yaml
          dart2js_args:
          - --dump-info
          - --fast-startup
          - --minify
          - --trust-type-annotations
          - --trust-primitives
```

**Comment by kevmoo** [comment]
You can configure the defaults using build.yaml - https://github.com/dart-lang/build/blob/master/docs/getting_started.md#switching-to-dart2js

**Comment by matanlurey** [comment]
Ah got it. Pull requests welcome...

**Comment by matanlurey** [comment]
Closed by #1218, thanks @Kleak!
